code/meta-parent/pom.xml

```diff
@@ -47,7 +47,7 @@
         </developer>
     </developers>
     <properties>
-        <jetty.version>9.2.11.v20150529</jetty.version>
+        <jetty.version>9.2.14.v20151106</jetty.version>
```

Comment:
@cjellick Is this a safe upgrade?  I know you had a lot of issues w/ Jetty and WebSockets.